lab-traceyr/gulpfile.js

```diff
@@ -0,0 +1,116 @@
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+var testFiles = ['test/**/*.js'];
+var appFiles = ['lib/**/*.js'];
+var gulpFiles = ['gulp.js'];
+
+gulp.task('lint:app', () => {
+  gulp.src(appFiles)
+.pipe(eslint({
```

Comment:
I'm not sure why the linter didn't pick this up, but there seems to be a spacing issue here